src/components/ChallangesPage.vue

```diff
@@ -1,21 +1,21 @@
 <template>
   <section class="main">
     <NavBar />
-    <br/>
-    <br/>
+    <br /><br />
+    <br /><br />
     <!-- <br/> -->
     <section class="teaam-title has-text-centered mt-3 mr-1 ml-1">
       <h1 class="title is-1 text-white mb-3 hvr hvr-underline-from-center">
         <span class="text-pink"> Challanges</span>
       </h1>
-<br/>
+      <br />
       <!-- <img
         class="soon-img has-text-centered"
         src="../assets/img/team_soon.png"
```

Comment:
Lets remove commented out code as well. If we're gonna need this in the future, add a TODO with your username,
Eg: `//TODO - @Suvink - <Reason for commenting out>`